### ISSUE TITLE
Consulta TPE

### ISSUE THREAD

**Issue by lcrosta** [PI-ITBA/2024_02#206]
Hola Marcelo :), una consulta. 
en la separacion del front y el back, el front es el que deberia corroborar que si es un archivo vacío, el archivo csv (de las queries) se cree vacio, no? Porque que pasaria si cambiaran el front, el backend no sabria si es vacio, cierto?

**Comment by marcelogarberoglio** [comment]
No tendría que ser el front el que haga algo distinto ni  tomarlo como un caso especial

Los pasos bien podrían ser:
1 Crear el TAD
2 Llenarlo con los datos 
3 Crear el archivo csv con los encabezados
4 Agregarle el resultado de la query
5 Cerrar el archivo

En los pasos 2 y 4 no se haría nada, pues no hay datos.

**Comment by ssommerr** [comment]
Claro, pero de alguna forma el front le deberia avisar al back que es algo vacio lo que se ha ingresado, podria el front al pasarle los datos estar ""notificandole esto de alguna forma, es decir en funcion a como se los pasa? El problema es que si otra persona hace el front y decide pasarlos de otras formas, no sabriamos como el back podria darse cuenta de lo que le estan pasando es nulo

**Comment by marcelogarberoglio** [comment]
No. Me parece que lo están encarando mal.
Pensalo como un ListADT o cualquier otro TAD que hayan hecho. 
Si crean el listADT, pero nunca llaman a la función add, y luego usan toBegin, hasnext, next. ¿Tuvieron que hacer algo especial para decirle al TAD "no hay datos"?
No importa si el front lo hace otra persona, en todos los casos va a ser lo mismo, cuando le pidan al TAD datos almacenados simplemente no habrá datos.

**Comment by ssommerr** [comment]
Okey genial, muchisimas gracias!
